packages/frontend/app/services/channels/vote-verifier.ts

```diff
@@ -0,0 +1,46 @@
+import Service, { inject as service } from '@ember/service';
+import VoteChain from 'emberclear/models/vote-chain';
+import CryptoConnector from '../workers/crypto';
+import WorkersService from '../workers';
+import VoteSorter from 'emberclear/services/channels/vote-sorter';
+
+export default class VoteVerifier extends Service {
+  @service workers!: WorkersService;
+  @service voteSorter!: VoteSorter;
```

Comment:
this injection should be `@service('channels/vote-sorter') voteSorter!: VoteSorter;`

services that are namespaced need their full path specified as an arg to `@service`

Reply:
fixed